apps/public-docsite-v9/src/Concepts/WebComponentsInterop/UsingFluentReactWithWebComponents.stories.mdx

```diff
@@ -0,0 +1,148 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Concepts/Developer/Web Components Interop/Using Fluent React with Web Components" />
+
+# Using Fluent React with Web Componenets
+
+Fluent React v9's extensible architecture and use of web platform features like CSS custom properties
+allow it to be extended to work well with [Web Components](https://developer.mozilla.org/en-US/docs/Web/API/Web_components),
+particularly [shadow DOM](https://developer.mozilla.org/en-US/docs/Web/API/Web_components/Using_shadow_DOM).
+
+Where it is necessary to modify Fluent React's default behavior to better interoperate with Web Components, the
+implementation will live in the [Fluent UI Contrib repository](https://github.com/microsoft/fluentui-contrib).
```

Comment:
I think this part is a little unclear, if you think of something better feel free to ignore my suggestion.
```suggestion
It is necessary to modify Fluent React v9's default behavior to better interoperate with Web Components. The
implementation lives in the [Fluent UI Contrib repository](https://github.com/microsoft/fluentui-contrib).
```